fennel/client_tests/test_dataset.py

```diff
@@ -863,6 +879,11 @@ def pipeline_aggregate(cls, activity: Dataset):
                 Count(
                     window=Window("forever"), into_field=str(cls.total_ratings)
                 ),
+                Stddev(
+                    window=Window("3d"),
+                    of="rating",
+                    into_field=str(cls.std_rating_3d),
+                ),
             ]
```

Comment:
Can we add a 7d window too and ensure that the sd deviations are indeed different